packages/bitcore-wallet-client/src/lib/common/constants.ts

```diff
@@ -52,11 +52,22 @@ export const Constants = {
         minDecimals: 2,
       }
     },
+    xrp: {
+      toSatoshis: 1e6,
+      full: {
+        maxDecimals: 6,
+        minDecimals: 6,
+      },
+      short: {
+        maxDecimals: 6,
+        minDecimals: 2,
+      }
+    },
     usdc: {
       toSatoshis: 1e6,
       full: {
-        maxDecimals: 8,
-        minDecimals: 8,
+        maxDecimals: 6,
```

Comment:
ETH related change, Is this causing an issue in the app now?

Reply:
reverted